features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/h2.sql

```diff
@@ -1287,12 +1287,22 @@ CREATE TABLE IF NOT EXISTS IDN_ORG_USER_INVITE_ROLE_ASSIGNMENT(
     FOREIGN KEY (APPLICATION_ID) REFERENCES SP_APP(UUID) ON DELETE CASCADE
 );
 
-CREATE TABLE IF NOT EXISTS IDN_ORG_USER_ASSOCIATION(
```

Comment:
Why did you remove this?

Reply:
Mistakenly removed when resolving conflicts.
fixed with https://github.com/wso2/carbon-identity-framework/pull/4904/commits/de7f9a97c7f701696ad692435fc471125e647f71